### PR TITLE
fix system collection creation on '.anon' user for 1.3

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q | A
| --- | ---
| Bug fix? | yes
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| License | MIT

#### What's in this PR?

Backport #2923 to 1.3

#### Why?

Needed.

### CODE REVIEW

File: src/Sulu/Component/Media/SystemCollections/SystemCollectionManager.php [hunk 1]
```diff
@@ -137,6 +138,10 @@ private function getUserId()
             return;
         }
 
+        if (!$token->getUser() instanceof UserInterface) {
```
Comment (danrot) [inline src/Sulu/Component/Media/SystemCollections/SystemCollectionManager.php]: Why not extracting `$token->getUser()` to `$user` beforehand?

**Comment by danrot** [comment]
@alexander-schranz What's the easiest way to test this?

File: src/Sulu/Component/Media/SystemCollections/SystemCollectionManager.php (hunk 1 above)
Comment (trickreich) [inline src/Sulu/Component/Media/SystemCollections/SystemCollectionManager.php]: @danrot The same code is now in Sulu 1.4

**Comment by alexander-schranz** [comment]
@danrot I think run a command which access a none exist system collection.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
 CHANGELOG for Sulu
 ==================
 
+* 1.3.8 (2017-01-25)
```
Comment (danrot) [inline CHANGELOG.md]: Please remove the date here, I'll do that on the release.

File: src/Sulu/Component/Media/SystemCollections/SystemCollectionManager.php (hunk 1 above)
Comment (danrot) [inline src/Sulu/Component/Media/SystemCollections/SystemCollectionManager.php]: Ok, it's fine then.

**Pull request merged**